pip/index.py

```diff
@@ -1019,7 +1019,7 @@ def fmt_ctl_no_use_wheel(fmt_ctl):
     fmt_ctl_no_binary(fmt_ctl)
     warnings.warn(
         '--no-use-wheel is deprecated and will be removed in the future. '
-        ' Please use --no-binary :all: instead.', DeprecationWarning,
+        ' Please use --no-binary :all: instead.', RemovedInPip9Warning,
```

Comment:
Already removed in pip9 ? It seems a little early for me. But of course it also depends on pip9 release date.
cc @dstufft